packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -100,7 +100,7 @@ export class VaModal {
   /*
    * Style of modal alert - info, error, success, warning
    */
-  @Prop() status?: 'continue' | 'error' | 'info' | 'success' | 'warning';
+  @Prop({ reflect: true }) status?: 'continue' | 'error' | 'info' | 'success' | 'warning';
```

Comment:
This prop was not being rendered as an attribute in storybook anymore, added reflect:true to fix chromatic discrepancies.